package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "dcrdex",
+  "version": "1.1.0",
+  "description": "[![Build Status](https://github.com/decred/dcrdex/workflows/Build%20and%20Test/badge.svg)](https://github.com/decred/dcrdex/actions) [![ISC License](https://img.shields.io/badge/license-Blue_Oak-007788.svg)](https://blueoakcouncil.org/license/1.0.0) [![GoDoc](https://img.shields.io/badge/go.dev-reference-blue.svg?logo=go&logoColor=lightblue)](https://pkg.go.dev/decred.org/dcrdex)",
+  "private": true,
+  "scripts": {
+    "lint": "markdownlint docs/wiki/*.md && markdownlint docs/release-notes/*.md"
```

Comment:
Eh, this doesn't seem right.  I don't think we should make an npm package just to run a linter, especially not at the root of the repo and named "dcrdex".

I guess I kinda implied we'd need some kind of CI solution, but I just run markdownlint one of two ways: (a) VSCode extension, or (b) system-wide install of markdownlint-cli.

Let's figure out enforcement of markdownlint policy later, and just use our tentative policy we've discussed for the fuzzing markdown.